Gemfile

```diff
@@ -7,7 +7,7 @@ source 'https://rubygems.org'
 gem 'va_common', '~> 0.4.1'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.6'
+gem 'rails', '4.2.7'
```

Comment:
gem 'rails', '~> 4.2.7'  would be better; there was a vulnerability in 4.2.7 of activerecord that caused tests to fail—resolved here: https://github.com/department-of-veterans-affairs/gi-bill-comparison-tool/pull/393/commits/5952f602f773f590d471b5f3975cb03e21787b41